File: mbtsdk/src/main/java/core/bluetooth/MbtAudioBluetooth.java
package core.bluetooth;


import android.content.Context;

abstract class MbtAudioBluetooth extends MbtBluetooth implements BluetoothInterfaces.IAudioBluetooth {

    MbtAudioBluetooth(Context context, BluetoothProtocol protocol, MbtBluetoothManager mbtBluetoothManager) {
        super(context, protocol, mbtBluetoothManager);
    }

}

File: mbtsdk/src/main/java/core/bluetooth/MbtAudioBluetooth.kt
package core.bluetooth

import core.bluetooth.BluetoothInterfaces.IAudioBluetooth

class MbtAudioBluetooth {
  companion object{
    var instance : ExtraBluetooth? = null
  }
  abstract class ExtraBluetooth internal constructor(
      protocol: BluetoothProtocol?,
      mbtBluetoothManager: MbtBluetoothManager) :

      MbtBluetooth(
          protocol,
          mbtBluetoothManager),

      IAudioBluetooth
}